air/src/trace/snapshots/miden_air__trace__tests__all_chiplet_column_ranges.snap

```diff
@@ -0,0 +1,34 @@
+---
+source: air/src/trace/tests.rs
+expression: layout
+---
+ALL CHIPLET COLUMN RANGES
+================================
+
+Chiplet Selector Counts:
+  - Hasher selectors: 1
+  - Bitwise selectors: 2
+  - Memory selectors: 3
+  - ACE selectors: 4
+  - Kernel ROM selectors: 5
+
+Hasher Chiplet:
+  Trace offset: 52
+  Selector range: 52..55
```

Comment:
Maybe more generally, when we output a range, it would be nice to also show the length of the range, like
```suggestion
  Selector range: 52..55 (width 3)
```